artemis-quorum-api/src/main/java/org/apache/activemq/artemis/quorum/MutableLong.java

```diff
@@ -0,0 +1,51 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.activemq.artemis.quorum;
+
+public interface MutableLong extends AutoCloseable {
```

Comment:
Can you called this as MutableSequence, since it seems to be used on Sequence Management on Distributed Lock?


or if you don't want to change it.. can you add some javadoc on how this is used please? Perhaps both?

Reply:
It's a general purpose counter that allow `long` values: on Curator is called AtomicDistributedLong, but here I allow non-atomic ops, that's why I've called it MutableLong.